user-interface/src/data-verification/ConsolidationOrderModal.tsx

```diff
@@ -148,21 +186,88 @@ function ConsolidationOrderModalComponent(
     }
   }
 
+  function resizeModal() {
```

Comment:
Is this something we should have in the Modal component? We'll never want a modal that extends beyond the viewport.

Reply:
Maybe. Probably. Would need to assess how to make this easily generalizable.

Reply:
Fine for now. If we encounter this again we need to enhance the Modal component to handle it's own window resize listener hook.